Tools/sitl_multiple_run.sh

```diff
@@ -43,7 +43,7 @@ while [ $n -le $sitl_num ]; do
 
 	pushd "$working_dir" &>/dev/null
 	echo "starting instance $n in $(pwd)"
-	sudo -b -u $user ../src/firmware/posix/px4 -d "$src_path" rcS >out.log 2>err.log
+	sudo -b -u $user ../bin/px4 -i $n -d "$src_path" -s rcS >out.log 2>err.log
```

Comment:
sudo?

Reply:
I was wondering about that too. I just kept it the same now.